diagrams/request_uri_mode_create.md

```diff
@@ -0,0 +1,49 @@
+```plantuml
+@startuml
+
+autonumber
+
+box "Wallet"
+participant "Metadata" as wm
+participant "Authorization Endpoint" as w
+end box
+
+participant "User Agent" as u
+
+box "Verifier"
+participant "Frontend" as r
+participant "Request Endpoint" as rp
+participant "Response Endpoint" as rb
+end box
+
+u --> r : use
+activate r
+
+r --> u: authorization request\n(client_id, request{request_uri, request_uri_mode=POST, state})
```

Comment:
isn't the state parameter provided in the request object obtained from the request_uri?

Reply:
it's contained in both, where the state from the authorization request is sent to the request URI. This allows the verifier to identify the origin/context of the request URI request. It then may add a state to the new request object, too.

Reply:
> it may add a state to the new request object too

ok, points below needs clarifications:

- may it have the state only in the request object and not in the url params?
- what happen if the state is in both places but with different values?